### PR TITLE
pkp/pkp-lib#2135 Password reset process update

### PR DESCRIPTION
**Pull request by touhidurabir**
This PR aims to solve https://github.com/pkp/pkp-lib/issues/2135

### CODE REVIEW

File: locale/da_DK/emails.po [hunk 1]
```diff
@@ -32,19 +32,8 @@ msgstr ""
...
-msgstr ""
-"Din adgangskode er blevet nulstillet og kan benyttes på webstedet "
-"{$siteTitle}. Opbevar brugernavnet og adgangskoden, da de er nødvendige for "
-"alt arbejde i forbindelse med tidsskriftet.<br />\n"
-"<br />\n"
-"Dit brugernavn: {$recipientUsername}<br/>\n"
-"Adgangskode: {$password}<br />\n"
-"<br />\n"
-"Mvh. {$siteContactName}"
+msgid "emails.passwordResetConfirm.description"
```
Comment (asmecher) [inline locale/da_DK/emails.po:35]: @touhidurabir, I think this is an accidental re-add of an obsolete locale key.

File: locale/ms_MY/emails.po [hunk 2]
```diff
@@ -120,6 +120,105 @@ msgstr ""
 "Terima kasih, <br />\n"
 "{$signature}"
 
+msgid "emails.reviewerRegister.subject"
```
Comment (asmecher) [inline locale/ms_MY/emails.po:123]: Are these adds intentional?

File: locale/da_DK/emails.po (hunk 1 above)
Comment (touhidurabir) [reply locale/da_DK/emails.po]: Removed.

File: locale/ms_MY/emails.po (hunk 2 above)
Comment (touhidurabir) [reply locale/ms_MY/emails.po]: That was an accidental merge for that one specific file during rebase . Removed it .

File: locale/ms_MY/emails.po [hunk 3]
```diff
@@ -120,6 +120,102 @@ msgstr ""
 "Terima kasih, <br />\n"
 "{$signature}"
 
+#, fuzzy
+msgid "emails.userValidateContext.body"
```
Comment (asmecher) [inline locale/ms_MY/emails.po:124]: I think these additions are unintentional.

Comment (touhidurabir) [reply locale/ms_MY/emails.po]: Yes, these got into during the rebase, my mistake. Fixed it . Also update the few more locale files where the missing locales(`emails.passwordReset.subject` and `emails.passwordReset.body`) added after the rebase .

**Review by asmecher — APPROVED** [review]

**Comment by asmecher** [comment]
Cherry-picked manually due to submodule collision.

**Pull request closed without merging**